src/crimson/os/seastore/cache.h

```diff
@@ -499,24 +500,7 @@ class Cache : public ExtentTransViewRetriever {
       } else {
         // stable from trans-view
         assert(!p_extent->is_pending_in_trans(t.get_trans_id()));
-        if (t.maybe_add_to_read_set(p_extent)) {
```

Comment:
Two worries here:
1. It also delays mutation-pending stable extents which is unrelated to this issue.
2. There are may other places calling `*add_to_read_set(extent)` in Cache, I don't see why all the other places are unrelated to the issue.